cfg/rationalize.go

```diff
@@ -38,5 +38,9 @@ func Rationalize(c *Config) error {
 		return err
 	}
 
+	if c.Debug.Fuse || c.Debug.Gcs || c.Debug.LogMutex {
```

Comment:
Same method is also there in config_util.go (OverrideWithLoggingFlags).
Let's refactor/cleanup the code to use the common logic.